src/browser/components/molecules/TemplateTaskForm/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import InputField from '../../atoms/InputField'
+import Button from '../../atoms/Button'
+
+export default class TemplateTaskForm extends Component{
+
+  render() {
+    return(
+      <div>
+        <form onSubmit={(event)=>this.props.submit(event)}>
+          <label>
+            <InputField label='name' name='name' update={(event, inputField)=>this.props.update(event, inputField)} />
```

Comment:
Also the benefit of that is that, you dont need to pass in the `event` as an argument.